compiler/il/OMRNode_inlines.hpp

```diff
@@ -207,27 +207,28 @@ OMR::Node::getChild(int32_t c)
 inline TR::Node *
 OMR::Node::getFirstChild()
    {
-   TR_ASSERT(self()->getNumChildren() >= 1, "getFirstChild() called on node %p with no children", this);
+   TR_ASSERT(self()->getNumChildren() >= 1, "getFirstChild() called on node " POINTER_PRINTF_FORMAT " with no children", this);
```

Comment:
Since `TR::Node` is an extensible class, `self()` is more appropriate rather than `this`.

This comment applies to your other modifications below too.